### PR TITLE
[Swift4] Add throw to reserved words

### PR DESCRIPTION
**Pull request by atanasbozhkov**
### PR checklist

- [X] Read the [contribution guidelines](https://github.com/swagger-api/swagger-codegen/blob/master/CONTRIBUTING.md).
- [X] Ran the shell script under `./bin/` to update Petstore sample so that CIs can verify the change. (For instance, only need to run `./bin/{LANG}-petstore.sh` and `./bin/security/{LANG}-petstore.sh` if updating the {LANG} (e.g. php, ruby, python, etc) code generator or {LANG} client's mustache templates). Windows batch files can be found in `.\bin\windows\`.
- [X] Filed the PR against the correct branch: `3.0.0` branch for changes related to OpenAPI spec 3.0. Default: `master`.
- [X] Copied the [technical committee](https://github.com/swagger-api/swagger-codegen/#swagger-codegen-technical-committee) to review the pull request if your PR is targeting a particular programming language.

### Description of the PR
Adding `throw` in the list of Swift 4 reserved words. This is to address #6919 
(details of the change, additional tests that have been done, reference to the issue for tracking, etc)

I have simply added the word to the hash-set and regenerated the Petstore samples.
I haven't added any changes to the Swift4 tests - currently they seem to only test a single word as a pass condition. Should there be some integration tests that iterate through the set values to see if all reserved words are escaped properly?
 Lastly - it's my first time contributing to the project - please let me know if I've skipped a step in the PR process - I'll do my best to fix it.

### CODE REVIEW

**Comment by atanasbozhkov** [comment]
Technical Committee:
@jgavris  @ehyche  @Edubits  @jaz-ah

**Comment by wing328** [comment]
> Should there be some integration tests that iterate through the set values to see if all reserved words are escaped properly?

We don't at the moment but of course we welcome more tests to cover different edge cases.

File: samples/client/petstore/swift4/default/.swagger-codegen/VERSION [hunk 1]
```diff
@@ -1 +1 @@
-2.3.0-SNAPSHOT
\ No newline at end of file
+unset
```
Comment (wing328) [inline samples/client/petstore/swift4/default/.swagger-codegen/VERSION]: I wonder why this is `unset`...

**Comment by wing328** [comment]
@atanasbozhkov if you've time, I wonder if you can do the same to Swift3Codegen.java.

**Comment by atanasbozhkov** [comment]
@wing328 - yes I actually thought about that today. I'll look at the integration tests as well - any suggestions where they should go?

**Comment by wing328** [comment]
https://github.com/swagger-api/swagger-codegen/tree/master/samples/client/petstore/swift4/default/SwaggerClientTests/SwaggerClientTests should be a good starting point.
